tests/test_diffusion.py

```diff
@@ -0,0 +1,188 @@
+from parcels import Field, Grid, JITParticle, ScipyParticle, Variable, SaptiallyVaryingDiffusion2D, random, ErrorCode
```

Comment:
I still think this function is more an 'example' than a unit-test, so I think it would be better placed in the `examples` directory (and be renamed `example_SpatiallyVaryingDiffusion2D.py`